client/components/first-view/index.jsx

```diff
@@ -43,7 +43,9 @@ const FirstView = React.createClass( {
 	},
 
 	componentWillUnmount() {
-		this.updateDocumentStylesForHiddenFirstView();
+		process.nextTick( () => {
```

Comment:
`process.nextTick` is a shim provided by [node-process](https://github.com/defunctzombie/node-process) [included in our webpack config](https://github.com/Automattic/wp-calypso/blob/7414620edccc9741b7a6642e16a3dc1ee39322dc/webpack.config.js#L85).

I'm happy to switch this if you both still agree given that context, but it's more clear to me what `process.nextTick( fn )` does as opposed to `setTimeout( fn, 0 )`